app/src/main/java/com/android/unio/model/strings/test_tags/HomeTestTags.kt

```diff
@@ -0,0 +1,13 @@
+package com.android.unio.model.strings.test_tags
+
+object HomeTestTags {
+  const val MAP_BUTTON = "event_MapButton"
+  const val SCREEN = "HomeScreen"
+  const val HEADER = "event_Header"
+  const val TAB_ALL = "event_tabAll"
+  const val TAB_SAVED = "event_tabFollowing"
+  const val UNDERLYING_BAR = "event_UnderlyingBar"
+  const val SEARCH_BAR_INPUT = "searchBarInput"
+  const val SEARCH_BAR = "searchBar"
```

Comment:
Why not follow the convention of having event_ as a prefix for the search bar? Or if the prefix doesn't serve a purpose, why have it for the other test tags?

Reply:
You're absolutely correct, I haven't paid attention to that, I simply extracted all the hardcoded strings. But as said below, this extraction will help make these types of changes easily! 
I will look into defining these strings better now before merging!